src/marker/with_marker.es

```diff
@@ -0,0 +1,9 @@
+import React, { forwardRef } from 'react';
+import { Consumer } from './context';
+
+
+module.exports = (Component) => (
```

Comment:
why commonjs? also why not simply use `Consumer` directly, why do you need  a hoc?

Reply:
Context is changing